spec/request/api/v1/targets/create_spec.rb

```diff
@@ -24,16 +24,44 @@
       it 'is expected that response contains some body data' do
         subject
         body = JSON response.body
-        decimal_scale = 6
+        decimal_scale = 10
         expect(json_value(body, 'target', 'id')).not_to be_nil
         expect(json_value(body, 'target', 'user_id')).not_to be_nil
         expect(json_value(body, 'target', 'area_lenght')).to eq params[:target][:area_lenght]
         expect(json_value(body, 'target', 'title')).to eq params[:target][:title]
         expect(json_value(body, 'target', 'topic')).to eq params[:target][:topic]
-        expect(json_value(body, 'target', 'latitude').to_d.round(decimal_scale))
-          .to eq params[:target][:latitude].to_d.round(decimal_scale)
-        expect(json_value(body, 'target', 'longitude').to_d.round(decimal_scale))
-          .to eq params[:target][:longitude].to_d.round(decimal_scale)
+        expect(json_value(body, 'target', 'latitude').round(decimal_scale))
+          .to eq params[:target][:latitude].round(decimal_scale)
+        expect(json_value(body, 'target', 'longitude').round(decimal_scale))
+          .to eq params[:target][:longitude].round(decimal_scale)
+      end
+
+      context "when the created target is compatible with other target's user" do
```

Comment:
Here you're testing a success case with only one target. It'd be better if you can create a state more complex with 
2 targets compatibles and 1 no compatible. Also with different users too. You can use `share context` if you think this can be reusable.